src/app/core/shell/toolbar/toolbar.component.ts

```diff
@@ -23,8 +23,8 @@ import { OrganizationService } from 'app/organization/organization.service';
 })
 export class ToolbarComponent implements OnInit {
 
-  /** Eligible permissions to view the country dropdown and an admin menu. */
-  allowedPermissions: any = ["ALL_FUNCTIONS"];
+  /** Eligible roles to view the country dropdown and an admin menu. */
+  allowedRoles: any = ["super user"];
```

Comment:
If we're going to have multiple roles here, can you use a `Set` instead? Just so we can benefit from O(1) lookup